pkg/kubectl/cmd/drain.go

```diff
@@ -613,16 +620,21 @@ func (o *DrainOptions) evictPods(pods []corev1.Pod, policyGroupVersion string, g
 	for {
 		select {
 		case err := <-errCh:
-			return err
```

Comment:
Do we not now lose the underlying reason? Looking below I see we just do `"Drain did not complete..."`.

Reply:
Since there can be N different errors, they get glogged and counted. A summary error is returned on line 635. Do you think we should capture all the error messages?

Reply:
outputting via glog like this rather than actually returning errors means DrainOptions can't easily be used programmatically or composed into larger commands